scripts/emerge/one_step_data_ingest.py

```diff
@@ -0,0 +1,57 @@
+import argparse
+import json
+import pandas as pd
+
+
+def create_recoded_json(row_json):
+    """Update dictionary with TDR's dataset relative paths for keys with gs:// paths."""
+
+    all_row_jsons = []
+    for key in row_json.keys():
+        value = str(row_json[key])  # convert to string to be able to check if gs:// path
+
+        # if value exists and has 'gs://'
+        if value is not None and "gs://" in value:
+            relative_tdr_path = value.replace("gs://","/")  # create TDR relative path
+
+            # # TODO: add in description = id_col + col_name
+            row_json[key] = {"sourcePath":value,
+                             "targetPath":relative_tdr_path,
+                             "mimeType":"text/plain"
```

Comment:
I wasn't even setting this (I'm not sure why they ask or what the effect is)… and why TDR isn't reading mimeType off the imported file itself from GCS

Reply:
im not sure either but last time i left it out i was told it was required so i just added it in